app/models/hyrax/resource.rb

```diff
@@ -103,6 +103,17 @@ def visibility
       visibility_reader.read
     end
 
+    def embargo=(value)
+      raise TypeError "can't convert #{value.class} into Hyrax::Embargo" unless value.is_a? Hyrax::Embargo
+
+      @embargo = value
+      self.embargo_id = @embargo.id
+    end
+
+    def embargo
+      @embargo ||= (Hyrax.query_service.find_by(id: embargo_id) if embargo_id.present?)
```

Comment:
Are the extra parenthesis necessary here? 

```suggestion
      @embargo ||= Hyrax.query_service.find_by(id: embargo_id) if embargo_id.present?
```

Reply:
nope. this looks to have been left over from trying some other things.